index.js

```diff
@@ -99,6 +99,18 @@ const doRemovePrompts = ( token, repo ) => {
         });
       };
 
+const removeAllLabels = ( token, repo ) => {
+  octonode.client(token).get('/repos/' + repo + '/labels', (e, status, body) => {
+    let allLabels = body.map((label) => {
+      return { name: label.name, color: label.color };
+    });
+
```

Comment:
@Jameskmonger my only bit of code review here would be to add a conditional to see if there are even any labels to remove. If there aren't any, we can just `console.log("No labels to remove")` and not run `gitLabel`, which is going to try to make that external API call to delete 0 labels.

Thanks for contributing! :smile: